### PR TITLE
Add manager resource

### PR DESCRIPTION
**Pull request by ZePedroResende**

### CODE REVIEW

File: priv/repo/migrations/20180724140327_create_staff.exs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+defmodule Safira.Repo.Migrations.CreateStaff do
+  use Ecto.Migration
+
+  def change do
+    create table(:attendees) do
```
Comment (0urobor0s) [inline priv/repo/migrations/20180724140327_create_staff.exs]: Shouldn't this be staff?

File: lib/safira/accounts/staff.ex [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  schema "staffs" do
+    field :active, :boolean
+
+    belongs_to :user, User
+
+    timestamps()
+  end
+
+  def changeset(attendee, attrs) do
+    attendee
```
Comment (0urobor0s) [inline lib/safira/accounts/staff.ex]: Shouldn't this be staff?

File: lib/safira/accounts/staff.ex [hunk 3]
```diff
@@ -0,0 +1,21 @@
+defmodule Safira.Accounts.Staff do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Safira.Accounts.User
+
+  schema "staffs" do
```
Comment (0urobor0s) [inline lib/safira/accounts/staff.ex]: The plural is staff

Comment (ZePedroResende) [reply lib/safira/accounts/staff.ex]: True but i think for readability sake we should keep it as staffs

File: priv/repo/migrations/20180724140327_create_staff.exs (hunk 1 above)
Comment (ZePedroResende) [reply priv/repo/migrations/20180724140327_create_staff.exs]: Damm mistakes were made :$

File: lib/safira/accounts/staff.ex (hunk 3 above)
Comment (0urobor0s) [reply lib/safira/accounts/staff.ex]: I think it's known to be staff no need for extra s

File: config/dev.exs [hunk 4]
```diff
@@ -8,7 +8,7 @@ use Mix.Config
 # with brunch.io to recompile .js and .css sources.
 config :safira, SafiraWeb.Endpoint,
   http: [port: 4000],
-  debug_errors: true,
+  debug_errors: false,
```
Comment (0urobor0s) [inline config/dev.exs]: They help in dev

**Pull request merged**